client/backend.js

```diff
@@ -188,6 +188,10 @@ module.exports = {
     },
 
     updateAccess(accessId, access) {
+        if (access.customFields) {
+            access.customFields = JSON.stringify(access.customFields);
```

Comment:
Can you use a ternary conditional here (operator `?`), and move this down to the data passed ton the ajax call, as it's done in createAccess?